scylla/src/transport/connection.rs

```diff
@@ -1,9 +1,9 @@
 use bytes::Bytes;
```

Comment:
Regarding the commit message:

> If data is available, it's immediately processed as well and kept in a buffer in order to be sent in a single write_all() call.

I don't think it's 100% true after introducing `BufWriter`.

Reply:
yup, it got out of date, thanks. Fixed